Fitness/Controllers/GymDetailViewController.swift

```diff
@@ -36,16 +36,20 @@ class GymDetailViewController: UIViewController {
     var starButton: UIButton!
     var gymImageView: UIImageView!
     var titleLabel: UILabel!
+    
+    // Exists when gym.isOpen is false
+    var closedLabel: UILabel?
 
     var hoursTitleLabel: UILabel!
     var hoursTableView: UITableView!
     var days: [Days]!
     var hoursData: HoursData!
     var hoursBusynessSeparator: UIView!
 
-    var popularTimesTitleLabel: UILabel!
-    var popularTimesHistogram: Histogram!
-    var busynessFacilitiesSeparator: UIView!
+    // Exist when gym.isOpen is true
+    var popularTimesTitleLabel: UILabel?
+    var popularTimesHistogram: Histogram?
+    var busynessFacilitiesSeparator: UIView?
```

Comment:
nit: typo

Reply:
For once that typo was actually intentional: I felt that `businessFacilitiesSeparator` could be kind of ambiguous, that said I'll replace it with `popularTimesFacilitiesSeparator` since that's both actually clear and properly spelled lol